tests/lib/tools/tests.info

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -eu
+
+show_help() {
+    echo "usage: tests.info <CMD>"
+    echo
+    echo "Supported commands:"
+    echo "    is-snapd-pkg-repo: indicates when the snapd package is from the repository"
```

Comment:
I'm sorry to nitpick about this but this doesn't read like English. Can we try something like `is-snapd-installed-from-system-archive`. It's longer but it doesn't need explanation.

Reply:
perhaps is-snapd-from-system-archive ?

Reply:
updated

Reply:
Thank you!